src/Control/Carrier/Trace/Printing.hs

```diff
@@ -1,4 +1,8 @@
 {-# LANGUAGE FlexibleInstances, GeneralizedNewtypeDeriving, MultiParamTypeClasses, TypeOperators, UndecidableInstances #-}
+
+{- | A carrier for the 'Control.Effect.Trace' effect that prints all traced results to stderr.
```

Comment:
💭 Maybe we should rename this carrier’s module to `.Stderr` (out of scope for this PR).